moveit_ros/moveit_servo/src/collision_monitor.cpp

```diff
@@ -98,6 +98,9 @@ void CollisionMonitor::checkCollisions()
   double self_collision_scale, scene_collision_scale;
   const double log_val = -log(0.001);
 
+  // Get a read-only copy of the planning scene.
+  planning_scene_monitor::LockedPlanningSceneRO locked_scene(planning_scene_monitor_);
```

Comment:
I didn't see a need to have to instantiate this every time in a loop, so I moved it up. Please correct me if I'm misunderstanding something here.